### PR TITLE
Fully implemented CRUD for users including FE Integration

### PR DESCRIPTION
**Pull request by RyanBirtch-aot**
Create Read Update Delete Fully implemented for users. Added jest test cases for service improvements.

### CODE REVIEW

File: frontend/.env [hunk 1]
```diff
@@ -1 +1 @@
-REACT_APP_BACKEND_API="http://localhost:3001"
\ No newline at end of file
+REACT_APP_BACKEND_API="http://localhost:3010"
```
Comment (acoard-aot) [inline frontend/.env:1]: This file should likely be ignored and removed from git, to stop accidentally leaking secrets.

We can .gitignore it, and it's also good practice to create an `example.env` file which has all the fields (but not values) to make first time setup easier.

**Review by acoard-aot — CHANGES_REQUESTED** [review]
Looks good, only thing that jumped out was the frontend/.env file needing to be ignored/secret.  Made a few other comments too but in general looks great, we should be able to merge all PRs in today

File: frontend/.env (hunk 1 above)
Comment (midhun-aot) [reply frontend/.env]: We can discuss on it and for now we will not check-in the secrets in .env for dev as well

File: backend/applications/src/app.module.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    //   realm: 'epd-dev',
+    //   clientId: 'applicationapi',
+    //   secret: 'cNrfJNtuyCije5Em5I0oIJA1oFV9IsxQ',
+    //   policyEnforcement: PolicyEnforcementMode.PERMISSIVE,
+    //   // Secret key of the client taken from keycloak server
+    // }),
+        ApplicationModule,
+    TypeOrmModule.forRoot({
+      type: 'postgres',
+      host: process.env.POSTGRESQL_HOST || 'gldatabase',
```
[the remark this answers is not in the document]
Comment (midhun-aot) [reply backend/applications/src/app.module.ts]: Great

File: backend/applications/.gitignore [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# compiled output
```
[the remark this answers is not in the document]
Comment (midhun-aot) [reply backend/applications/.gitignore]: Yes, we can work on this later

**Review by acoard-aot — APPROVED** [review]
Great stuff!

**Review by acoard-aot — APPROVED** [review]
Woohooooo! So many envs.

**Pull request merged**